docs/src/modules/sandbox/CodeSandbox.ts

```diff
@@ -24,10 +24,11 @@ function openSandbox({ files, codeVariant, initialFile }: any) {
   form.target = '_blank';
   form.action = 'https://codesandbox.io/api/v1/sandboxes/define';
   addHiddenInput(form, 'parameters', parameters);
+  addHiddenInput(form, 'embed', '1');
   addHiddenInput(
     form,
     'query',
-    `file=${initialFile}${initialFile.match(/(\.tsx|\.ts|\.js)$/) ? '' : extension}`,
+    `module=${initialFile}${initialFile.match(/(\.tsx|\.ts|\.js)$/) ? '' : extension}`,
```

Comment:
embed has changed from `file` to `module`. There is no documentation, I have to get the embed to know that it's changed.

Eg:
```
<iframe src="https://codesandbox.io/embed/9sc876?view=editor+%2B+preview&module=%2FSignIn.tsx"
     style="width:100%; height: 500px; border:0; border-radius: 4px; overflow:hidden;"
     title="recursing-rain-9sc876"
     allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
     sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
   ></iframe>
```